src/components/v2v3/shared/SplitItem.tsx

```diff
@@ -143,13 +170,28 @@ const SplitValue = ({
         {splitValueFormatted}
         {valueSuffix ? <span> {valueSuffix}</span> : null})
       </span>
+      <div
+        style={{
+          fontSize: '.8rem',
+          color: colors.text.secondary,
+          marginLeft: 10,
+        }}
+      >
+        {showFees && !isJuiceboxProject && (
+          <Parenthesis>
+            <CurrencySymbol currency={curr} />
+            {t` ${feeAmountFormatted} fee`}
```

Comment:
When marking for translation, we want to translate the entire 'sentence' because some languages have different sentence structures.

```suggestion
          <Trans>
            <CurrencySymbol currency={curr} />{' '}
            {feeAmountFormatted} fee
            </Trans>
```